frontend/src/features/rollout-announcement/components/NewFeatureTag.tsx

```diff
@@ -0,0 +1,19 @@
+import { Icon, Text } from '@chakra-ui/react'
+
+import { BxsRocket } from '~assets/icons'
+import Badge from '~components/Badge'
+
+export const NewFeatureTag = (): JSX.Element => {
+  return (
+    <Badge
+      variant="subtle"
+      display="inline-flex"
+      columnGap="0.5rem"
+      alignItems="center"
+      color="secondary.500"
+    >
+      <Icon as={BxsRocket} />
+      <Text textStyle="caption-1">New feature</Text>
```

Comment:
nit: don't think textStyle is required here. the base component has the text style already included.
nit: don't think you even need to wrap the text in a `Text` component.